packages/@ember/-internals/glimmer/lib/helpers/eq.ts

```diff
@@ -0,0 +1,30 @@
+import { Arguments, CapturedArguments, VM } from '@glimmer/runtime';
+import { InternalHelperReference } from '../utils/references';
+
+/**
+@module ember
+*/
+
+/**
+  Compares the two given values with ===
+
+  Example:
+
+  ```handlebars
+  {{eq type "button"}}
+
+  {{! be true if `type` is "button"}}
+  ```
+
+  @public
```

Comment:
The docs all need `@category ember-basic-template-helpers` (this _should_ allow the docs to avoid shipping to emberjs.com/api/ before we’ve enabled the feature flag)...